### PR TITLE
chore: Adding v0.0.17 signed windows provisioning scripts to VHD builds

### PR DESCRIPTION
**Pull request by marosset**
Signed-off-by: Mark Rossetti <marosset@microsoft.com>

**Reason for Change**:

Includes new signed windows provisioning scripts

**Issue Fixed**:

**Credit Where Due**:

Does this change contain code from or inspired by another project?

- [x] No
- [ ] Yes

If "Yes," did you notify that project's maintainers and provide attribution?

- [ ] No
- [ ] Yes

**Requirements**:

- [x]  uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [x] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: vhd/packer/configure-windows-vhd.ps1 [hunk 1]
```diff
@@ -113,7 +113,8 @@ function Get-FilesToCacheOnVHD {
             "https://kubernetesartifacts.azureedge.net/aks-engine/windows/provisioning/signedscripts-v0.0.13.zip",
             "https://kubernetesartifacts.azureedge.net/aks-engine/windows/provisioning/signedscripts-v0.0.14.zip",
             "https://kubernetesartifacts.azureedge.net/aks-engine/windows/provisioning/signedscripts-v0.0.15.zip",
-            "https://kubernetesartifacts.azureedge.net/aks-engine/windows/provisioning/signedscripts-v0.0.16.zip"
+            "https://kubernetesartifacts.azureedge.net/aks-engine/windows/provisioning/signedscripts-v0.0.16.zip",
+            "https://kubernetesartifacts.azureedge.net/aks-engine/windows/provisioning/signedscripts-v0.0.17.zip"
```
Comment (jackfrancis) [inline vhd/packer/configure-windows-vhd.ps1:117]: Any reason to take this opportunity to cull this list down to just the latest (or maybe the last few)?

File: vhd/packer/configure-windows-vhd.ps1 [hunk 2]
```diff
@@ -203,8 +204,7 @@ function Install-WindowsPatches {
             # then you can get download links by searching for specific KBs at http://www.catalog.update.microsoft.com/home.aspx
 
             # Find a specific patch at https://www.catalog.update.microsoft.com/Search.aspx?q=kb5005625
-            # KBKB5015880 Contains 2022 7C patches for Windows server 2019
-            $patchUrls = @("https://catalog.s.download.windowsupdate.com/c/msdownload/update/software/updt/2022/07/windows10.0-kb5015880-x64_1b8eadaa8f12dacd77342353c257e236e167a802.msu")
+            $patchUrls = @()
```
Comment (jackfrancis) [inline vhd/packer/configure-windows-vhd.ps1:207]: Is this deletion related to the new signedscripts package, or just general cleanup?

Comment (marosset) [reply vhd/packer/configure-windows-vhd.ps1:207]: No, this deletion is related to https://github.com/Azure/aks-engine/pull/4978/files#diff-70e0f93cdd528a80723f7809a66d9f64a323e66e162393d144007318a7c09a46R5

We were previously installing on of the `7C` patch which is the optional updates from July. 
I updated the base image we use for the packer build to include the newest security patches (which are newer than these) so we don't need to install the optional updates anymore.

File: vhd/packer/configure-windows-vhd.ps1 (hunk 1 above)
Comment (marosset) [reply vhd/packer/configure-windows-vhd.ps1:117]: Eh, these are so small compared to the rest of the content on the images...
We didn't remove versions when the project was active so I'm not really seeing a need to do so now.

Comment (jackfrancis) [reply vhd/packer/configure-windows-vhd.ps1:117]: sgtm

**Review by jackfrancis — APPROVED** [review]
/lgtm

**Pull request merged**